guide/glossary.md

```diff
@@ -0,0 +1,319 @@
+
+Differentiate between
+
+- Bitcoin wallet account
+- Service account
+
+## Bitcoin / bitcoin
+
+Bitcoin with a capital *B* is typically associated with Bitcoin the protocol and payment network. It is also often used to refer to as the ecosystem as a whole when writing about it in general terms. Bitcoin with a lowercase “b” written as “bitcoin” is usually associated specifically with bitcoin as the currency.
+
+## Bitcoin Core
+
+A Bitcoin wallet often considered as a *reference implementation* of the Bitcoin protocol. It is the continuation of Satoshi Nakamoto's original code. More on the [Bitcoin Wiki](https://en.bitcoin.it/wiki/Bitcoin_Core).
+
+## BIP - Bitcoin improvement proposal
+
+A standardized design document format for suggesting improvements to Bitcoin. They are hosted on Github [here](https://github.com/bitcoin/bips){:target="_blank"}. Some important proposals to be aware of:
+
+- [BIP39](https://github.com/bitcoin/bips/blob/master/bip-0039.mediawiki){:target="_blank"}: Mnemonic code for generating deterministic keys
+- [BIP44](https://github.com/bitcoin/bips/blob/master/bip-0044.mediawiki){:target="_blank"}: Multi-account hierarchy for HD wallets
+- [BIP49](https://github.com/bitcoin/bips/blob/master/bip-0049.mediawiki){:target="_blank"}: Derivation scheme for HD wallets using P2WPKH-nested-in-P2SH, nested SegWit
+- [BIP84](https://github.com/bitcoin/bips/blob/master/bip-0084.mediawiki){:target="_blank"}: Derivation scheme for HD wallets using P2WPKH, native SegWit
+- [BIP141](https://github.com/bitcoin/bips/blob/master/bip-0141.mediawiki){:target="_blank"}: Segregated Witness, changes to transaction structure
+- [BIP173](https://github.com/bitcoin/bips/blob/master/bip-0173.mediawiki){:target="_blank"}: *Bech32* standard for native segregated witness addresses 
+- [BIP174](https://github.com/bitcoin/bips/blob/master/bip-0174.mediawiki){:target="_blank"}: Partially Signed Bitcoin Transaction Format
+
+## Coin control
+
+<div class="center" markdown="1">
+
+{% include image.html
+   image = "/assets/images/guide/glossary/coin-selection.jpg"
+   retina = "/assets/images/guide/glossary/coin-selection@2x.jpg"
+   alt-text = "Illustration of specific coins a user has received getting chosen for a new transacton."
+   layout = "float-right-desktop"
+   width = 400
+   height = 400
+%}
+
+The act of choosing which coins (really unspent outputs, or UTXOs) to forward to another address in a transaction. Wallet-applications can automatically choose the coins to use, but there are scenarios when users may want to manually choose what coins to send.
```

Comment:
```suggestion
The act of choosing which UTXO to forward to another address in a transaction. Wallet-applications can automatically choose the coins to use, but there are scenarios when users may want to manually choose what coins to send.
```

Reply:
Can we use 'unspent transaction' or something more readable? 
UTXO needs its own explanation. (I did not write this myself though so not an expert here)

Reply:
I will commit the suggestion and add 'unspent transaction output' and link to the UTXO explanation.